crawler/helpers.py

```diff
@@ -576,9 +577,9 @@ class AggregateType22(AggregateTypeBase):
     def __init__(self):
         super().__init__()
         self.type_str = "TYPE 22"
-        self.error_level = ErrorLevel.CRITICAL
+        self.error_level = ErrorLevel.ERROR
         self.message = (
-            "CRITICAL: Files where the DART database inserts have failed for some plates. "
+            "ERROR: Files where the DART database inserts have failed for some plates. "
```

Comment:
```suggestion
            f"{self.error_level.name}: Files where the DART database inserts have failed for some plates. "
```

Reply:
Just thought of this now...

Reply:
I'm going to not include this change, for the sake of getting this merged in and ready for UAT deployment by the end of the day